app/views/twilio/call.haml

```diff
@@ -1,3 +1,3 @@
 !!! XML
-%Response
-  %Dial= @user.phone
+%response
```

Comment:
Not sure if XML tags are case-sensitive, or how permissive Twilio is to this if so, is why I ask.